src/ui.js

```diff
@@ -25,8 +25,8 @@ export const makeTable = async () => {
     row.push(
       company[COMPANY_NAME_FIELD_NAME],
       company[STATUS_FIELD_NAME],
-      company[CREATED_AT_FIELD_NAME],
-      company[REVENUE_YTD_FIELD_NAME],
+      new Date(company[CREATED_AT_FIELD_NAME]).toLocaleString('en', {dateStyle: 'short', timeStyle: 'short', hour12: false}),
+      company[REVENUE_YTD_FIELD_NAME].toString().replace(/(\d)(?=(\d{3})+(?!\d))/g, "$1 "), 
```

Comment:
Regular expressions are great in some cases, but they are also really hard to understand by a person who is reading your code. For example, me - I have no idea what it does. This is not great for code maintainability. Therefore, I would recommend in the future consider avoiding those and use different implementation that is easier to read by other team members.

Reply:
@DataDrivenEngineer Thank you for the feedback. I will certainly keep this in mind going forward.